scripts/lis_hyperv_platform/main.ps1

```diff
@@ -20,6 +20,7 @@ param(
     [String] $QemuPath = "C:\bin\qemu-img.exe",
     [String] $UbuntuImageURL = "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img",
     [String] $CentosImageURL = "https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2",
+    [String] $BootLogPath="C:\$InstanceName-boot.log",
```

Comment:
there is no need to specify another param -- use JobId (JobPath) for the base-folder